.github/workflows/deploy_to_prod_lambda.yml

```diff
@@ -0,0 +1,35 @@
+name: Deploy to liveClassAttendanceHandler Lambda
```

Comment:
I'll mention all my comments here:

The process we follow for other repositories and I think is a good approach is this:
There will be two github actions. One will deploy to staging lambda and one will deploy to production lambda.

Now thee times when these should be triggered should also be different.
When someone makes a PR, OR merges a PR to the main branch, then the staging github action should run and deploy to staging.
When someone pushes code into the release branch, then the prod github action should run and deploy to prod.

and this is what our flow is, make a PR, the code gets deployed to staging. We test it out and then merge it into main, at which point the code gets deployed again to staging so staging is up to date.

Now when we're confident that this code will go to production, we'll merge the main branch into the release branch, and at that point of time the code will get deployed to production

Reply:
Also, this branch is created off of the previous one right? the IP address one? If that is the case then please change the base branch for this PR to the IP address one instead of main because then it shows the IP address changes in this PR as well

Reply:
So I agree with two github actions for staging and prod. That's the process in Portal also. But the issue for this repo is there is no local/dev environment. Whenever I have to develop a feature/change, I work on staging only. So that's pushed from locally. So I didn't find the need for another action in this case.

Reply:
Understood. But currently you're deploying to staging everytime from your local right? That is another manual step which is prone to errors. If you had a github action for that, you can just commit your code to a PR and it will deploy it for you. But that's up to you how to do it.

And I would suggest to setup a python file `run_locally.py` so you don't have to push the code to staging lambda everytime you wanna test. You can and should test locally first. You can see [this](https://github.com/avantifellows/etl-data-flow/blob/master/flows/assessments/run_locally.py) to see an example of run locally function

Reply:
The thing about having a github action for staging and commiting my code to PR is, when I'm building something, there'll be a lot of commits in the process.

What do you mean by pushing code to staging everytime I want to test? That is what is happening, no? The only solution to this is to have other dev lambda, but I don't see the point, since it'll be a paid resource.

Reply:
Another dev lambda will be free, because it would come under our monthly free tier of lambda so that won't be an issue. But yeah more trouble than it's worth right now so let it be.
Just make sure you create a `run_locally.py` using which anyone testing can try to run the lambda function locally